internal/policygen/iam.go

```diff
@@ -85,3 +126,26 @@ func validate(instance map[string]interface{}, mandatoryFields []string) error {
 	}
 	return nil
 }
+
+func projectNumber(rn runner.Runner, id string) (string, error) {
+	cmd := exec.Command("gcloud", "projects", "describe", id, "--format", "json")
+	out, err := rn.CmdOutput(cmd)
+	if err != nil {
+		return "", fmt.Errorf("failed to get project number for project %q: %v", id, err)
+	}
+
+	type project struct {
+		ProjectNumber string `json:"projectNumber"`
+	}
+
+	var p project
```

Comment:
Nit: Can do `var p struct { ... }`

Reply:
`p` is actually to be passed to the `Unmarshal` function to hold the struct parsed from `out`.

Reply:
Done